CONTRIBUTING.md

```diff
@@ -214,8 +214,12 @@ When a person is assigned to an issue this indicates an _intent to implement_.
 
 * Include screenshots and animated GIFs in your pull request whenever possible.
 * List any steps necessary to trigger the feature you've created or bug you are fixing
-* Always run the [tests](#tests) locally before creating your PR
+* Always run the [unit tests](#unit-tests) locally before creating your PR
+ * You are not required to run the [integration tests](#integration-tests) locally
+* Once the tests have passed in the PR you must receive a review using the GitHub review system
+ * To learn more about GitHub reviews take a look at their [documentation](https://help.github.com/articles/reviewing-changes-in-pull-requests/) and [video tutorial](https://youtu.be/HW0RPaJqm4g)
 * Request review from @jasonLaster or @jlongster by mentioning their names in the PR
+ * Also mention either of their names in the PR if you're having issues with [integration tests](#integration-tests) passing
```

Comment:
Yeah, this seems a little redundant. Another bullet point probably isn't required, but you can add to the the other one.